src/components/EnvironmentOverride/DeploymentTemplateOverride.tsx

```diff
@@ -441,6 +456,13 @@ export default function DeploymentTemplateOverride({
                         isEnterpriseInstallation={
                             currentServerInfo?.serverInfo?.installationType === InstallationType.ENTERPRISE
                         }
+                        isValuesOverride={isValuesOverride}
+                        setIsValuesOverride={setIsValuesOverride}
+                        groupedData={groupedOptionsData}
+                        manifestDataRHS={manifestDataRHS}
```

Comment:
We should create all these in the reducer. Why are we mixing the useState and useReducer states

Reply:
resolved